src/configure-picasso.js

```diff
@@ -17,6 +18,7 @@ export default function configurePicasso() {
   picasso.component('reference-line-labels', refLineLabelsComponent);
   picasso.component('point-label', pointLabelComponent);
   picasso.component('disclaimer', disclaimer());
+  picasso.component('custom-rect', customRect);
```

Comment:
Can probably give this a more descriptive name. Since the custom component is located only in scatter-plot, there is no need to give it a generic name like 'custom-rect', so can probably just rename it the same as what it is used for in scatter, 'background-rect', or 'mini-map-window' or something like that.